SU2_CFD/src/output/CFlowCompOutput.cpp

```diff
@@ -35,7 +35,7 @@ CFlowCompOutput::CFlowCompOutput(CConfig *config, unsigned short nDim) : CFlowOu
 
   turb_model = config->GetKind_Turb_Model();
   lastInnerIter = curInnerIter;
-  gridMovement = config->GetGrid_Movement();
+  gridMovement = (config->GetGrid_Movement() || config->GetDynamic_Grid());
```

Comment:
Just noticed in another PR that for this you only need Dynamic_Grid as it already contains Grid_Movement